new/src/ReportFilter.test.ts

```diff
@@ -23,29 +24,29 @@ describe("Report filter status filter", () => {
         ]
     };
     test("should filter by status", () => {
-        jest.spyOn(repository, "getReport").mockReturnValue(fullReport);
+        vitest.spyOn(repository, "getReport").mockReturnValue(fullReport);
 
         expect(filterByStatus(ScenarioStatusFilter.SUCCESS)).toEqual([
             fullReport.scenarios[0].scenarios[0]
         ]);
     });
 
     test("should ignore undefined status", () => {
-        jest.spyOn(repository, "getReport").mockReturnValue(fullReport);
+        vitest.spyOn(repository, "getReport").mockReturnValue(fullReport);
```

Comment:
I always thought that that function is in the `vi` object. 
But lets not change that here...